tests/streamlit_ui_tests/grid/set_column_formula_tests.spec.ts

```diff
@@ -286,3 +288,77 @@ test('Write spreadsheet formula applied to individual cell', async ({ page }) =>
     const cellValues = await getValuesInColumn(mito, columnHeader);
     expect(cellValues).toEqual(['1', '0', '0', '0']);
 });
+
+test('Cross-sheet formula with VLOOKUP', async ({ page }) => {
```

Comment:
One more test to that since we had a bug in this before is: Start writing a cross sheet formula using the cell editor and then once you write `=VLOOKUP(column10,` switch to the formula bar to finish the formula